setupext.py

```diff
@@ -595,13 +593,11 @@ def do_custom_build(self):
                  "--with-png=no", "--with-harfbuzz=no"],
                 env=env, cwd=src_path)
             subprocess.check_call(["make"], env=env, cwd=src_path)
-        else:
-            # compilation on windows
-            shutil.rmtree(str(pathlib.Path(src_path, "objs")),
-                          ignore_errors=True)
+        else:  # compilation on windows
+            shutil.rmtree(str(Path(src_path, "objs")), ignore_errors=True)
```

Comment:
```suggestion
            shutil.rmtree(Path(src_path, "objs"), ignore_errors=True)
```
as of Python 3.6 `rmtree` supports path-like objects.

Reply:
ok